src/components/RoomHeaderAvatars.tsx

```diff
@@ -1,63 +1,58 @@
-import PropTypes from 'prop-types';
 import React, {memo} from 'react';
 import {View} from 'react-native';
-import _ from 'underscore';
 import useStyleUtils from '@hooks/useStyleUtils';
 import useThemeStyles from '@hooks/useThemeStyles';
 import Navigation from '@libs/Navigation/Navigation';
 import CONST from '@src/CONST';
 import ROUTES from '@src/ROUTES';
+import type {Icon} from '@src/types/onyx/OnyxCommon';
 import Avatar from './Avatar';
-import avatarPropTypes from './avatarPropTypes';
 import PressableWithoutFocus from './Pressable/PressableWithoutFocus';
 import Text from './Text';
 
-const propTypes = {
-    icons: PropTypes.arrayOf(avatarPropTypes),
-    reportID: PropTypes.string,
+type RoomHeaderAvatarsProps = {
+    icons: Icon[];
+    reportID: string;
```

Comment:
```suggestion
    icons?: Icon[];
    reportID?: string;
```

...or if they are supposed to be required, you have to remove the default values on line 18.